demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1201,3 +1206,64 @@ def test_IsEntityNameContainExcludedWordValidator(
     assert len(results) == expected_number_of_failures
     if results:
         assert results[0].message == expected_error_message
+
+@pytest.mark.parametrize(
+    "content_items, old_content_items, expected_number_of_failures, expected_msgs",
+    [
+        (
+            [create_metadata_object(paths=["name"], values=["changedName"]),create_metadata_object()],
+            [create_metadata_object(), create_metadata_object()],
+            1,
+            ["Pack name was changed from HelloWorld to changedName, please undo."],
+        ),
+    ],
+)
+def test_ValidPackNameValidator_is_valid(content_items, old_content_items, expected_number_of_failures, expected_msgs):
+    """
+    Given:
+    content_items (Pack).
+        - Case 1: A Pack content item that its name has changed.
+        - Case 2: A Pack content item that its name hasn't changed.
+
+    When:
+        - Calling the PackNameValidator is_valid function.
+
+    Then:
+        - Make sure the right amount of pack name failed, and that the right error message is returned.
+        - Case 1: Should fail.
+        - Case 2: Shouldn't fail.
+    """
+    create_old_file_pointers(content_items, old_content_items)
+    results = PackNameValidator().is_valid(content_items)
+    assert len(results) == expected_number_of_failures
+    assert all(
+        [
+            result.message == expected_msg
+            for result, expected_msg in zip(results, expected_msgs)
+        ]
+    )
+    
+@pytest.mark.parametrize(
+    "content_item, expected_id, expected_fix_msg",
+    [
+        (
+            create_metadata_object(paths=["name"], values=["changedName"]),
+            "changedName",
+            "Changing pack name back to changedName.",
```

Comment:
```suggestion
            "HelloWorld",
            "Changing pack name back to HelloWorld.",
```
I think you tested with the opposite values of what you need.
Maybe in the `fix_message` we can add the name that it changed from (`Changing pack name back from {0} to {1}.`).